toolkit/imageconfigs/marketplace-gen2.json

```diff
@@ -0,0 +1,65 @@
+{
+    "Disks": [
+        {
+            "PartitionTableType": "gpt",
+            "MaxSize": 600,
```

Comment:
600MB is fine for the initial version so we can get started on the pipeline side of things, but we should discuss how much empty space we want there to be without specifying --os-disk-size-gb